new/rfcXXX-lifecycle-and-release-policies.md

```diff
@@ -0,0 +1,203 @@
+
+    As a Chef maintainer,
+    I want to be able to release the exact build that I have tested,
+    so I do not need to make an additional release build.
+
+    As a Chef user,
+    I want to know what the lifecycle policy for software is,
+    so I can can understand how the software I consume is managed.
+
+    As a Chef user,
+    I need Chef OSS version numbers to give me expectations about the extent of changes
+    so I can estimate my upgrade efforts.
+
+## Specification
+
+### Lifecycle Policies
+
+This RFC breaks down the Chef OSS lifecycle into three categories:
+
+  * [Cadence Release](#cadence-release)
+  * [Ad-hoc Release](#ad-hoc-release)
+  * [Maintenance Mode](#maintenance-mode)
+
+Each component that falls under the purview of the [Chef Maintainers](https://github.com/chef/chef/blob/master/MAINTAINERS.md) falls into one these categories. Which category software falls under will be reflected by the appropriate badge on the top of its `README.md` with a link back to this RFC.
+
+#### Cadence Release
+
+[![](https://img.shields.io/badge/OSS%20Lifecycle-Cadence%20Release-brightgreen.svg)](https://github.com/chef/chef-rfc/blob/master/rfcXXX-versioning-and-release-policy.md#cadence-release)
+
+Chef OSS that falls under Cadence Release is under heavy development by the community and Chef Software and is released on a regular cadence. Cadence Releases help ensure that Chef Maintainers get new features and minor bug fixes out to users in a timeline fashion while not overloading the users with numerous releases and monotonically growing version numbers.
+
+Because of the high velocity of development, it is possible for many new features to be added during any given release cycle. To keep the version numbers of [Stable Releases](#stable-release) meaningful, the **MINOR** version will increment by one for each Stable Release. This means that many new features may be added between two Stable Releases, but the **MINOR** version will only increment by one.
+
+Because so much can change from release to release, maintainers will select a `MAJOR.MINOR.PATCH` build as a [Release Candidate](#release-candidate). Maintainers will follow a slightly modified [Release Communication](#release-communication) procedure to make an announcement to the "chef-dev" [Discourse](https://discourse.chef.io) category, notifying users of the impending Stable Release. The maintainers will then wait a pre-determined amount of time to allow user to perform further testing. If no critical regressions or other release blockers are identified, the Stable Release will proceed as scheduled.
+
+The following projects adhere to Cadence Release:
+
+| Chef OSS | MAJOR Release Cadence | MINOR Release Cadence | Release Candidate Delay |
+|---|---|---|---|
+| Chef Client | Yearly (April)[<sup>1</sup>](#footnotes) | Monthly (2nd Week) | 3 business days |
+| ChefDK | Yearly (April) | Monthly (4th Week)[<sup>2</sup>](#footnotes) | 3 business days |
+
+#### Ad-hoc Release
```

Comment:
This is somewhat confusing terminology since we use ad-hoc to internally just mean random one-off testing builds.  These should be named something like hotfix / bugfix / security fix.